### ISSUE TITLE
HyperGAN Crash 'CUDA_ERROR_LAUNCH_FAILED'

### ISSUE THREAD

**Issue by kirbykirby56** [HyperGAN/HyperGAN#251]
I had HyperGAN fail when I tried to move the window to another screen. It looks like it's an error somewhere in TensorFlow, but maybe that's an issue with how we call TF? I decided to write it here in the case taht it might help.
Here's the related cmd output:
`2020-02-15 09:22:22.020538: E tensorflow/stream_executor/cuda/cuda_event.cc:29] Error polling for event status: failed to query event: CUDA_ERROR_LAUNCH_FAILED: unspecified launch failure
2020-02-15 09:22:22.026183: F tensorflow/core/common_runtime/gpu/gpu_event_mgr.cc:273] Unexpected Event status: 1`
(everything above this message is your normal loss info, nothing out of the ordinary)

**Comment by kirbykirby56** [comment]
I've just encountered the same issue, this time not within a window move. 

`2020-02-15 14:35:59.868125: E tensorflow/stream_executor/cuda/cuda_driver.cc:1035] could not synchronize on CUDA context: CUDA_ERROR_LAUNCH_FAILED: unspecified launch failure ::
2020-02-15 14:35:59.868129: E tensorflow/stream_executor/cuda/cuda_event.cc:29] Error polling for event status: failed to query event: CUDA_ERROR_LAUNCH_FAILED: unspecified launch failure
2020-02-15 14:35:59.901406: F tensorflow/core/common_runtime/gpu/gpu_event_mgr.cc:273] Unexpected Event status: 1
`

**Comment by martyn** [comment]
This should be fixed in 1.0 when released(uses pytorch).
